### PR TITLE
Fix MIME type on Resume files

### PR DESCRIPTION
**Pull request by SubJunk**

### CODE REVIEW

**Comment by SubJunk** [comment]
@valib does your Panasonic TV play "Resume" files currently? I had to update the MIME type in my renderer config to make it work so I wonder if the same change needs to be done to other Panasonic files

File: src/main/java/net/pms/dlna/DLNAResource.java [hunk 1]
```diff
@@ -772,6 +772,8 @@ public void addChild(DLNAResource child, boolean isNew, boolean isAddGlobally) {
 
 					if (resumeRes != null && resumeRes.media != null) {
 						resumeRes.media.setThumbready(false);
+						resumeRes.media.setMimeType(HTTPResource.VIDEO_TRANSCODE);
+						resumeRes.media.setThumbready(false);
```
Comment (Nadahar) [inline src/main/java/net/pms/dlna/DLNAResource.java]: @SubJunk Why set "thumbready" twice?

Comment (SubJunk) [reply src/main/java/net/pms/dlna/DLNAResource.java]: Good question, not sure how that happened. Will fix now

**Comment by SubJunk** [comment]
I resolved the issue that caused it to fail peer review so I'm merging

**Pull request merged**